tests/Javascript/Pages/Requestee/EditBookingForm.spec.js

```diff
@@ -0,0 +1,144 @@
+      reference:{"path":"9_1611689494_reference"},
+      onsite_contact: {}
+      }
+  
+     wrapper = shallowMount(EditBookingForm, {
+        localVue,
+        propsData: {
+          booking: booking
+      },
+    })
+    wrapper.vm.updateForm(booking)
+})
+
+
+
+test('toggleNullableForms()', () => {
+  const booking = {
+    event: {"start_event":"19:11","end_event":"19:13","title":"event 3","type":"event3","description":"3","guest_speakers":"3","attendees":"3","fee":"yes","music":"music","food":{"low_risk":false,"high_risk":false},"alcohol":false,"av":true,"furniture":true},
+    reservations:  [{"id":1,"room_id":8,"booking_request_id":1,"start_time":"2021-01-26T05:29:15.000000Z","end_time":"2021-01-26T05:31:15.000000Z","created_at":"2021-01-26T05:29:28.000000Z","updated_at":"2021-01-26T05:29:28.000000Z","room":{"id":8,"name":"event3","number":"3","floor":3,"building":"3","created_at":"2021-01-26T05:25:15.000000Z","updated_at":"2021-01-26T05:25:15.000000Z","status":"available","min_days_advance":null,"max_days_advance":null,"attributes":{"food":false,"sofas":null,"chairs":null,"tables":null,"alcohol":false,"computer":false,"projector":false,"fundraiser":false,"television":false,"whiteboard":false,"a_v_permitted":false,"ambiant_music":false,"coffee_tables":null,"sale_for_profit":false,"capacity_sitting":null,"capacity_standing":3},"room_type":"Mezzanine"}}],
+    reference:{"path":"9_1611689494_reference"},
+    onsite_contact: {},
+    // this.form.event.show?.contact === false
+
+    }
+
+   wrapper = shallowMount(EditBookingForm, {
+      localVue,
+      propsData: {
+        booking: booking
+    },
+  })
+  wrapper.vm.toggleNullableForms()
+})
+
+test('uploads can be set via a method', () => {
+  wrapper.vm.uploadedFiles([]);
+
+  expect(wrapper.vm.form.files).toStrictEqual([]);
+})
+
+test('disabling question forms will save their state in the form', () => {
+  wrapper.vm.form.event.show = {contact:false, fee: false, music: false}
+
+  wrapper.vm.toggleNullableForms();
+
+  expect(wrapper.vm.form.event.show).toStrictEqual({contact:false, fee: false, music: false});
+})
+
+test('filter returns date', () => {
+
+  expect(wrapper.vm.only_date(moment())).toBe(moment().format("dddd, Do MMMM YYYY"));
+})
+
+test('filter returns time', () => {
+
+  expect(wrapper.vm.only_time(moment())).toBe(moment().format("LT"));
+})
+
+// test('computed reservation should match res[0]', () => {
```

Comment:
If this test is no longer relevant it should be removed instead of commented-out